.github/reusable-build/action.yml

```diff
@@ -0,0 +1,20 @@
+name: Reusable action which builds struct2tensor
+
```

Comment:
It's also possible for an action to respond to the `workflow_call:` event: here, for example, https://github.com/scikit-image/scikit-image/blob/main/.github/workflows/wheels_recipe.yml gets called by https://github.com/scikit-image/scikit-image/blob/main/.github/workflows/wheel_tests_and_release.yml. It's more or less the same thing, though, because I can't see the difference between the inputs for both types of events.

Reply:
Yep, this is what I suggested in a related project but others voted for this approach. I'm fine either way, there doesn't seem to be much difference functionally speaking, even though the triggering mechanism seems like it is completely different.

Reply:
I just had a closer look, and I think the difference is only aesthetic: one gets a dropdown in the GitHub Actions UI where one can look at the "called" workflow's jobs, which are separated with an indent from the rest of the jobs from the original workflow: https://github.com/scikit-image/scikit-image/actions/runs/11210366034

Reply:
Yeah, good point. Happily the dropdown allows you to pass arguments through the github UI, but you don't get that with `workflow_call`.